companies/dbregio.json

```
@@ -0,0 +1,27 @@
+{
+    "slug": "dbregio",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "DB Regio AG",
+    "runs": [
+        "bahn.de"
+    ],
+    "address": "Europa-Allee 70-76\n60486 Frankfurt am Main",
```

```suggestion
    "address": "Europa-Allee 70-76\n60486 Frankfurt am Main\nDeutschland",
```
fixed address format